test/cm/configure_fedora34.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash
+
+# sudo dnf install -y podman git perl-open.noarch python3-devel gcc libpq-devel python3.9-devel
+# git clone https://github.com/mfleader/snappy-data-server.git
+# sudo chown -R vagrant snappy-data-server
+# sudo chgrp -R vagrant snappy-data-server
```

Comment:
delete these lines ?

Reply:
I can remove `configure_fedora34.sh`. I added it to the repo as a reflex.